src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -81,12 +70,7 @@
     @Advanced
     @Argument(fullName = "indelSizeToEliminateInRefModel", shortName = "ERCIS", doc = "The size of an indel to check for in the reference model", optional = true)
     public int indelSizeToEliminateInRefModel = 10;
-    
-    /**
-     * Bases with a quality below this threshold will reduced too the minimum usable qualiy score (6).
-     */
-    @Argument(fullName = "base_quality_score_threshold", shortName = "bqst", doc = "Base qualities below this threshold will be reduced to the minimum (" + QualityUtils.MIN_USABLE_Q_SCORE + ")", optional = true)
-    public byte BASE_QUALITY_SCORE_THRESHOLD = PairHMM.BASE_QUALITY_SCORE_THRESHOLD;
```

Comment:
Yeah -- laura is right... we hoped that using the PCR error model with a lot more hostility would have killed these FPs, but I couldn't make it hostile enough so we put in a hard filter instead.

But would be good to confirm that (unless Laura has?) by running the benchmark set with it both off and on (exomes specifically IIRC).